director/api/private.py

```diff
@@ -1,3 +1,4 @@
-from flask import Blueprint
+from flask import Blueprint, request
```

Comment:
Since we don't need request here why import it?

Reply:
removed